collections/_article/progress-report-web-export-in-4-3.md

```diff
@@ -0,0 +1,281 @@
+---
+title: "Web Export in 4.3"
+excerpt: "With single-threaded builds and sample playback, it's now easier than ever to export your game to the Web with Godot 4.3. And more!"
+categories: ["progress-report"]
+author: Adam Scott
+image: /storage/blog/covers/progress-report-web-export-in-4-3.webp
+image_caption_title: "Catburglar"
+image_caption_description: "An open source game by @JohnGabrielUK and his team"
+date: 2024-05-13 17:00:00
+---
+
+Have you ever begun some type of work, and only after realized how little you actually knew?
+That happened to me during [the last <abbr title="Game Developers Conference">GDC</abbr>](https://godotengine.org/article/gdc-2024-retrospective/).
+
+A few months ago, [Fabio Alessandrelli (@faless)](https://github.com/Faless/) and I switched positions:
+I took over his responsibilities as [Web Platform Lead](https://godotengine.org/teams/#platforms) for Godot to reduce
+his task load.
+
+I'm well used to the Web and its quirks. I began creating websites in
+<abbr title="Extensible Hypertext Markup Language">XHTML</abbr> and (vanilla) JavaScript, back in the days.
+Flash games were my jam! Enough so that I even worked for a video game studio as an ActionScript 3 developer in 2010.
+And I didn't really stop caring about the Web platform ever since.
+
+Since then, Flash may have died, but online games didn't.
+
+# The new situation
+But what I realized at the <abbr title="Game Developers Conference">GDC</abbr> is that we're entering in sort into a
+Golden Age of Web games. Not only websites like [Poki](https://poki.com/) or
+[Crazy Games](https://www.crazygames.com/) are super popular, but big players are starting to integrate Web games into
+their services, such as
+[Discord Activities](https://support.discord.com/hc/en-us/articles/4422142836759-Activities-on-Discord)
+or [YouTube Playables](https://www.youtube.com/playables). And all these entities want developers to create games for
+their platform. And they are all asking how Godot could help create fast-loading "instant games".
+
+# Godot 4.3
+Making games for the Web using Godot can be a challenge, especially on 4.x, and that means serious revisions are needed.
+Godot 4.3 promises to be one of the best recent releases about Web exports. One of the biggest issues relating to this
```

Comment:
Just some clarification here. The experience in Godot 3.x is pretty good and its what we are aiming for with 4.x. And the problem is not making games per se, its really just the experience of shipping games on all platforms (particularly MacOS). When you are developing on Windows or linux the entire process is great until a Mac user tries to run your game. 

```suggestion
Making games for the Web using Godot 4.x still isn't as seamless as we would like it to be. Unfortunately serious revisions are needed to improve the experience to the extent that we want.
Godot 4.3 promises to be one of the best recent releases for Web exports. One of the biggest issues relating to this
```